docs/03-tutorials/00-eventing/evnt-03-type-cleanup.md

```diff
@@ -0,0 +1,255 @@
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+```bash
+cat <<EOF | kubectl apply -f -
+   apiVersion: eventing.kyma-project.io/v1alpha1
+   kind: Subscription
+   metadata:
+     name: lastorder-payment-sub
+     namespace: default
+   spec:
+     sink: 'http://lastorder.default.svc.cluster.local'
+     filter:
+       filters:
+         - eventSource:
+             property: source
+             type: exact
+             value: ''
+           eventType:
+             property: type
+             type: exact
+             value: sap.kyma.custom.myapp.order.payment-success.v1
+EOF
+```
+
+To check that the Subscription was created and is ready, run:
+```bash
+kubectl get subscriptions lastorder-payment-sub -o=jsonpath="{.status.ready}"
+```
+
+The operation was successful if the returned status says `true`.
+  </details>
+</div>
+
+## Check the Subscription cleaned Event type
+
+To check the Subscription cleaned Event type, run:
+```bash
+kubectl get subscriptions lastorder-payment-sub -o=jsonpath="{.status.cleanEventTypes}"
+```
+
+Now, note that the returned cleaned event type `["sap.kyma.custom.myapp.order.paymentsuccess.v1"]` does not contain the hyphen (i.e. `-`) in `payment-success` part, because Eventing services cleans out the non-alphanumeric characters from the Event name and uses the cleaned Event name in underlying Eventing-Backend.
+
+## Trigger the workload with an event
+
+We created the `lastorder` Function, and subscribed to the `order.payment-success.v1` events by creating a Subscription CR. 
+Now we will see that we can still publish events with the original Event name (i.e. `order.payment-success.v1`) even though it contains the non-alphanumeric character, and it will trigger the Function.
+In this example, we'll port-forward the Kyma Eventing Service to localhost.
+
+1. Port-forward the Kyma Eventing Service to localhost. We will use port `3000`. Run:
+   ```bash
+   kubectl -n kyma-system port-forward service/eventing-event-publisher-proxy 3000:80
+   ```
+2. Now publish an event to trigger your function. In another Terminal window run:
+
+   <div tabs name="Publish an event" group="trigger-workload">
+     <details open>
+     <summary label="CloudEvents Conformance Tool">
+     CloudEvents Conformance Tool
+     </summary>
+   
+      ```bash
+      cloudevents send http://localhost:3000/publish \
+         --type sap.kyma.custom.myapp.order.payment-success.v1 \
+         --id e4bcc616-c3a9-4840-9321-763aa23851fc \
+         --source myapp \
+         --datacontenttype application/json \
+         --data "{\"orderCode\":\"3211213\", \"orderAmount\":\"1250\"}" \
+         --yaml
+      ```
+   
+     </details>
+     <details>
+     <summary label="curl">
+     curl
+     </summary>
+   
+      ```bash
+      curl -v -X POST \
+           -H "ce-specversion: 1.0" \
+           -H "ce-type: sap.kyma.custom.myapp.order.payment-success.v1" \
+           -H "ce-source: /default/io.kyma-project/custom" \
```

Comment:
same as above: source attribute between curl and conformance tool are different.

Reply:
updated